endevor/README.md

```diff
@@ -6,6 +6,9 @@ This [sample repository](Automated-Test-Facility-for-Batch-Applications) contain
 ## Building a z/OS CICS Blockmode Application with Endevor®, Zowe, and Gen
 This [sample repository](../gen/gen-whitepaper-sample) contains the artifacts described in the [Building a z/OS CICS Blockmode Application with Endevor® and Zowe whitepaper](https://community.broadcom.com/mainframesoftware/communities/community-home/digestviewer/viewthread?GroupId=1513&MessageKey=7a3ba595-6432-48aa-93f4-f18206875d72&CommunityKey=4182c217-4789-4997-8f22-87de25983f6e&tab=digestviewer). There are Python scripts that you can use as-is or modify to better match your organization's DevOps practices. Also included are SCL templates (or skeletons) that show examples of processors you will need to have present in your organization's Endevor installation to successfully compile and link your Gen applications with Endevor.
 
+## Endevor - Self-servicing Project Workareas in Endevor with Dynamic Environments
```

Comment:
Because this entry is in the endevor README, the Endevor prefix should be removed